test/DynamoCoreWpfTests/InfoBubbleTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Dynamo.Tests;
+using Dynamo.ViewModels;
+using NUnit.Framework;
+using System;
+using System.Windows;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    public class InfoBubbleTests : DynamoViewModelUnitTest
+    {
+        [Test]
+        public void CanTriggerDocumentationLinkEvent()
+        {
+            // Arrange
+            InfoBubbleViewModel infoBubble = new InfoBubbleViewModel(this.ViewModel);
+            const string documentationFile = "dynamo.html";
+            string content = $"This is the test infoBubble href={documentationFile}";
+            InfoBubbleDataPacket infoBubbleDataPacket = new InfoBubbleDataPacket(InfoBubbleViewModel.Style.Error,
+                new Point(0, 0), new Point(0, 0), content, InfoBubbleViewModel.Direction.Bottom);
+            var docEventArgs = new OpenDocumentationLinkEventArgs(infoBubbleDataPacket.Link);
+
+            // register an extra handler for documentation events so we can confirm
+            // the Dynamo ViewModel event gets triggered from the InfoBubble viewmodel
+            RequestOpenDocumentationLinkHandler handler = (OpenDocumentationLinkEventArgs e) =>
+            {
+                Assert.IsNotNull(e);
+                Assert.AreEqual(documentationFile, e.Link.ToString());
+            };
+            this.ViewModel.RequestOpenDocumentationLink += handler;
```

Comment:
please unsubscribe this after the test.

Reply:
96e73795a3aceae6871b26f0cd828b09d5d747dc